internal/bundle.go

```diff
@@ -184,6 +187,7 @@ func (bundle *Bundle) CheckSizeAndEnqueueBack(tarBall TarBall) error {
 		bundle.mutex.Lock()
 		defer bundle.mutex.Unlock()
 
+		atomic.AddInt64(bundle.AllTarballsSize, tarBall.Size())
```

Comment:
Seems like this row should be added before each .CloseTar() call, not only in CheckSizeAndEnqueueBack(), but in uploadLabelFiles, UploadPGControl, FinishQueue too

Reply:
It wasn't necessary because I used current tarball size together with bundle.AllTarballssize 
```	uncompressedSize := bundle.TarBall.Size() + atomic.LoadInt64(bundle.AllTarballsSize)``` but it will work as well if we do it each time before closing tarball.

Reply:
I think it is a good idea to extract this increment to a method to avoid duplication

Reply:
yeah, I already pushed the change.

Reply:
fixed that.